### PR TITLE
[Executables] run_tests3.bat + disable __pycache__

### PR DESCRIPTION
**Pull request by gpotter2**

### CODE REVIEW

**Comment by p-l-** [comment]
Could you use names similar to the unix ones ("xxx.bat", "xxx_py2.bat", "xxx_py3.bat")?

**Comment by gpotter2** [comment]
@p-l- Updated

File: test/run_tests_py3 [hunk 1]
```diff
@@ -1,3 +1,4 @@
 #! /bin/sh
 PYTHON=python3
+PYTHONDONTWRITEBYTECODE="True" # Disable __pycache__
```
[the remark this answers is not in the document]
Comment (gpotter2) [reply test/run_tests_py3]: Ok

**Comment by gpotter2** [comment]
So should I add a run_scapy.bat.lnk file ? (That's a shortcut file, as symlinks are unusable on windows)

**Comment by p-l-** [comment]
@gpotter2 maybe a simple copy? BTW what happens to symlinks when you clone a repository (e.g., `README -> README.md` in Scapy's root)?

**Comment by gpotter2** [comment]
`README` is not-openable file containing a single line of text:
`README.md`

**Comment by gpotter2** [comment]
Update:
- fixed PYTHONDONTWRITEBYTECODE
- added run_scapy.bat, which selects either python 2 or python 3 if available (python 2 by default at least for now)
- updated run_scapy_py2 and run_scapy_py3 with a `--nopause` optional parameter, to manually force-disable the final PAUSE (which triggered only when scapy crashes)

**Pull request merged**